app/presenters/auction_presenter.rb

```diff
@@ -89,20 +89,24 @@ def bid_count
     bids.size
   end
 
-  def starts_at
+  def formatted_start_time
     DcTimePresenter.convert_and_format(model.start_datetime)
   end
 
-  def ends_at
+  def formatted_end_time
     DcTimePresenter.convert_and_format(model.end_datetime)
   end
 
-  def starts_in
+  def formatted_delivery_deadline
+    DcTimePresenter.convert_and_format(model.delivery_deadline)
+  end
+
+  def relative_start_time
     time_in_human(model.start_datetime)
   end
 
-  def ends_in
-    time_in_human(model.end_datetime)
+  def relative_time_left
+    "#{distance_of_time_in_words(Time.now, model.end_datetime)} left"
```

Comment:
I didn't know about that. It looks like it doesn't affect the calculations, so sure...